extension/src/FileFunctions.ts

```diff
@@ -12,7 +12,10 @@ export function findFiles(pattern: string, root: string): string[] {
   return fileList.sort((a, b) => a.localeCompare(b));
 }
 
-function getAllFilesRecursive(dir: string, fileList: string[] = []): string[] {
+export function getAllFilesRecursive(
+  dir: string,
+  fileList: string[] = []
```

Comment:
Why do we pass `fileList`? It really looks like it should be declared inside the function.